modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -26,6 +26,7 @@ The Schedules tab is especially useful for the following use cases:
 * While CloudHub supports polling frequencies in seconds, you may see discrepancies of a few seconds between individual polling calls. Setting a polling frequency higher than once in 10 seconds is not recommended.
 * If a scheduled job is missed (not triggered) one or more times because the application is down, CloudHub triggers the job once as soon as the application restarts.
 * Cloudhub scheduler reads a schedule job configuration every time it is run. This means that if you update the schedule time, the change is not immediate. The job will execute as it was already configured and only then the scheduler will pick up the next setting for the next run. To have your changes applied immediately, run your jobs after every schedule update.
+* Poll execution time should not take longer than 12 hours, otherwise the scheduler will be considered as failed and re executed.
```

Comment:
If poll execution time takes longer than 12 hours, the scheduler fails and is re-executed.